Mage/src/main/java/mage/abilities/effects/common/DoIfClashWonEffect.java

```diff
@@ -31,7 +32,7 @@ public DoIfClashWonEffect(Effect effect) {
     public DoIfClashWonEffect(Effect effect, boolean setTargetPointerToClashedOpponent, String chooseUseText) {
         super(Outcome.Benefit);
         this.executingEffect = effect;
-        this.chooseUseText = chooseUseText;        
+        this.chooseUseText = chooseUseText;
```

Comment:
Did something change here?

Reply:
My IDE did remove some trailing space.
If such code cleanup is not wanted, I can definitively tweak its options.

Reply:
All good